src/util/RoundRobinLB.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AbstractLoadBalancer} from './AbstractLoadBalancer';
+import {randomInt} from '../Util';
+import {Member} from '../core/Member';
+
+/**
+ * A {@link LoadBalancer} implementation that relies on using round robin
+ * to a next member to send a request to.
+ */
+export class RoundRobinLB extends AbstractLoadBalancer {
+    private index: number;
+
+    constructor() {
+        super();
+        this.index = randomInt(Date.now());
```

Comment:
Java client doesn't use randomization, but only monotonic clock available in the runtime. See:
https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/util/RoundRobinLB.java#L39

In case of Node.js, `process.hrtime()` function could be used:
https://nodejs.org/api/process.html#process_process_hrtime_time